test-requirements.txt

```diff
@@ -1,10 +1,10 @@
-pytest
-mock
-Flask
-Flask-CORS
-Flask-SQLAlchemy
-lxml
-requests
-pytest-stress
-locust
-python-dateutil
+Flask==2.0.3
```

Comment:
What do these versions represent? The latest available versions compatible with a specific Python version? The versions matching what is available in SLE for that Python version? Or something else...

It might be a good idea to provide a comment at the start of this file, and of the requirements.txt above, indicating the provenance of the versions being chosen.

Reply:
That's the lowest common denominator essentially, that it is the highest version that Python 3.6.15 can support.